src/models/wallet.models.ts

```diff
@@ -0,0 +1,55 @@
+import { z } from 'zod';
+
+export enum WalletKeys {
+  WALLET_TYPE = 'walletType',
+  WALLET_ADDRESS = 'walletAddress',
+  WALLET_NAME = 'walletName',
+  WALLET_FORMAT = 'walletFormat',
+  WALLET_OWNER = 'walletOwner',
+  AUTHENTICATION_TYPE = 'authenticationType',
+}
+
+export const PasskeyAuthenticationSchema = z.object({
+  challenge: z.string(),
+  attestation: z.object({
+    credentialId: z.string(),
+    clientDataJson: z.string(),
+    attestationObject: z.string(),
+    transports: z.array(z.string()),
+  }),
+});
+
+export const WalletSchema = z.object({
+  [WalletKeys.WALLET_ADDRESS]: z.string().optional(),
```

Comment:
because to reuse the same wallet type on different endpoints:
- auth type does not exist in the API response, just in the creation input data
- address does not exist at the moment of the creation either
- owner does not exist currently on the response for list wallets but was intended to